octobot_trading/personal_data/portfolios/types/future_portfolio.py

```diff
@@ -109,12 +128,14 @@ def update_portfolio_from_liquidated_position(self, position):
                 update_quantity = liquidated_quantity / position.mark_price
                 self._update_future_portfolio_data(position.currency,
                                                    position_margin_value=update_quantity,
-                                                   wallet_value=update_quantity)
+                                                   wallet_value=update_quantity,
+                                                   unrealized_pnl_value=-position.unrealised_pnl)
```

Comment:
?

Reply:
Here we are in `update_portfolio_from_liquidated_position` => to me we need either to apply the realized pnl or do this. In my tests we don't actually remove the liquidated amount from the portfolio